editor/test/effects.js

```diff
@@ -296,11 +296,11 @@ describe( 'effects', () => {
 			window.history.replaceState.mockRestore();
 		} );
 
-		const handler = effects.REQUEST_POST_UPDATE_SUCCESS;
-		const dispatch = jest.fn();
-		const store = { getState: () => {}, dispatch };
-
 		it( 'should dispatch meta box updates on success for dirty meta boxes.', () => {
+			const handler = effects.REQUEST_POST_UPDATE_SUCCESS;
+			const dispatch = jest.fn();
```

Comment:
We might have been able to share these variable references, since it appears the only thing we needed to do was ensure that `dispatch` calls were reset between tests when asserting call count.

https://facebook.github.io/jest/docs/en/mock-function-api.html#mockfnmockclear

Reply:
Yeah... I thought I changed this, but I guess I never did will patch now.

Reply:
Somehow this broke everything.